angular.json

```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "train-a": {
+      "projectType": "application",
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss",
+          "changeDetection": "OnPush"
+        }
+      },
+      "root": "",
+      "sourceRoot": "src",
+      "prefix": "app",
```

Comment:
Prefix should use product name, use `dd`, for example or `tt`. It could be `yt` for the previous youtube task, for example. Fix eslint accordingly.

Reply:
You could leave it as `app`, but please, discuss it with your team.